### DISCUSSION TITLE
Great work!! Where have you been this whole time?

### DISCUSSION THREAD

**Discussion by nacho-domatix** [tteck/Proxmox#514] [General]
I just found out this, and gotta say I'm quite impressed and amazed. I can' believe such an effort and this had been out there without the proper marketing. 

I just missed Nextcloud, any chance to see it too here?

It might help the [nextcloudpi](www.nextcloudpi.com) script, although I couldn't make it work in a Proxmox container.

**Comment by tteck** [comment]
Done!
